deep-learning/_posts/2018-11-08-dmri-connectivity-det-vs-prob.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: review
+title: "Mapping connectomes with diffusion MRI: deterministic or probabilistic tractography?"
+tags: deep-learning tractography brain white-matter
```

Comment:
no deep learning tag.
Also, should be put in some other folder than deep-learning